sretoolbox/container/image.py

```diff
@@ -150,6 +150,11 @@ def _get_auth(self, www_auth):
             auth = None
 
         response = requests.get(url, auth=auth)
+
+        if response.status_code == 401:
+            # Try again without auth
+            response = requests.get(url)
+
```

Comment:
Maybe we should try this **before** to avoid sending quay's credentials to docker.io.